minijinja/src/filters.rs

```diff
@@ -557,6 +557,23 @@ mod builtins {
         }
     }
 
+    /// Split a string with a separator
+    #[cfg_attr(docsrs, doc(cfg(feature = "builtins")))]
+    pub fn split(val: Value, split: Option<Cow<'_, str>>) -> Result<Value, Error> {
```

Comment:
I would change this filter in two ways so it's similar to the `.split` function in Python:

- if `None` or an empty string is passed (or nothing at all), it splits on any whitespace (`split_whitespace`).
- a second parameter (`nsplits`) is added which indicates how many times the string is to be split. `1` means one split (eg: two parts), `2` means two splits (eg: three parts).

Reply:
Alright - yeah, I was looking for your input before continuing. Sounds like a plan!